spec/features/purchases_spec.rb

```diff
@@ -19,45 +19,45 @@
   scenario 'display twenty purchases by default' do
     click_link 'Purchases'
 
-    numrows = page.all(:css, 'table tbody tr').size
+    purchases = page.all(:css, 'table tbody tr').size
 
-    expect(numrows).to be == 20
+    expect(purchases).to be == 20
   end
 
   scenario 'display five purchases after clicking appropriate button' do
     click_link 'Purchases'
     click_on 'button_limit_5'
 
-    numrows = page.all(:css, 'table tbody tr').size
+    purchases = page.all(:css, 'table tbody tr').size
```

Comment:
Ah, I steered you a little wrong here on naming - `purchase_count` would be more appropriate!